tests/e2e/scenarios/build/run-test.sh

```diff
@@ -23,10 +23,11 @@ make test-e2e-install
 
 cd "${GOPATH}"/src/k8s.io/kubernetes
 
+# this job assumes pull-kops-e2e-k8s-aws-calico is always ran on presubmits
```

Comment:
The marker is fixed now, this should not be needed. Expecting this to exist is also not quite correct.